app/scripts/services/acceptanceTest.js

```diff
@@ -106,11 +106,9 @@ angular.module('ludwig').factory('AcceptanceTestsService', function($q, $http, $
 
         launchTest: function(test) {
             test.running = true;
+
+            // Remove the test status during execution, so that the top bar of the test accordion is grey.
             delete test.status;
-            test.expectedResults.forEach(function(expectedResult) {
```

Comment:
Meh. Without delving deep into the code, I can't know for sure. Reading just this function, it seems like the removal should be fine. If you've tested it, it's fine by me.

Let's just poke @Flightan in case something comes up to his mind.